src/main/resources/META-INF/services/org.apache.pirk.responder.wideskies.spi.ResponderPlugin

```diff
@@ -0,0 +1,5 @@
+org.apache.pirk.responder.wideskies.mapreduce.MapReduceResponder
```

Comment:
My concern is that instead of keeping this code 'dumb' and forcing the user to give the specific ResponderLauncher implementation via the CLI (or props file), the PR seems to have moved to a service-plugin model where not only does the user have to specify the class/platform, the code has to be aware, via a static list, of the possible plugin implementations (as listed in org.apache.pirk.responder.wideskies.spi.ResponderPlugin). While this is far more flexible than the way that the platform selection was previously implemented, it does not remove the need for the codebase to understand (in one location) all of it's possible platforms. I am not opposed (as you pointed out, it is a well known model), but I thought that one of the primary goals was to remove the need for a static, central awareness of all possible platforms.